src/common/options.cc

```diff
@@ -7767,6 +7767,10 @@ static std::vector<Option> get_immutable_object_cache_options() {
     .set_default(false)
     .set_description("immutable object cache promote throttle"),
 
+    Option("immutable_object_cache_throttle_type", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
```

Comment:
This shouldn't be needed -- it should support both concurrently. Also -- it's an unrelated change from the commit message.

Reply:
sorry, i'm still confused about this comments. when one LOOKUP request arrived, how do we decide which throttle type to use ?  
Or,  librbd hook should firstly tell RO daemon which throttle type it use on register process ?

Reply:
If the user has specified a IOPS limit only -- use it. If the user has specified a bytes limit only -- use it. If a user has specified both -- use both. We do this in the IO path for librbd via multiple throttles (one for each). You always attempt to take from both throttles but only proceed once both throttles have credits available.

Reply:
hi, jason. i have refined all codes in this pull request based on your suggestions, and also modified commit messages. Additional, could i apply for access to Redhat's Teuthology lab ?